Translate OpenBSD architecture name to the name used by upstream.

Index: configure
--- configure.orig
+++ configure
@@ -19,6 +19,7 @@
     ('amd64','amd64'), ('x86_64','amd64'),
     ('x86','x86'), ('i386','x86'), ('i686','x86'),
     ('arm64','arm64'), ('armv8','arm64'), ('aarch64','arm64'),
+    ('armv7','arm32'),
     ('arm','arm32'),
     ('riscv64','riscv64'),
     ('riscv','riscv32'),
@@ -27,6 +28,7 @@
     ('ppc64','ppc64'), ('powerpc64','ppc64'),
     ('powerpc','ppc32'),
     ('ppc','ppc32'),
+    ('macppc','ppc32'),
     ('sparc64','sparc64'), ('sparcv9','sparc64'), ('sun4u','sparc64'),
     ('sparc','sparc32'), ('sun','sparc32'),
   ):
